modules/tinymce/src/core/main/ts/text/Whitespace.ts

```diff
@@ -1,8 +1,13 @@
 import { Arr, Strings, Unicode } from '@ephox/katamari';
 
+import { ZWSP } from './Zwsp';
```

Comment:
`Unicode` is already imported, and everything in the `Zwsp` module is an alias into the `Unicode` module.

- [ ] use `Unicode` directly

Reply:
done